content/en/cloud_cost_management/cost_allocation/bigquery.md

```diff
@@ -0,0 +1,112 @@
+---
+title: BigQuery Cost Allocation
+description: Learn how to allocate Cloud Cost Management spending across your organization with BigQuery Cost Allocation.
+further_reading:
+  - link: "/cloud_cost_management/"
+    tag: "Documentation"
+    text: "Learn about Cloud Cost Management"
+---
+
+## Overview
+
+Datadog Cloud Cost Management (CCM) automatically allocates the costs of your Google BigQuery resources to individual queries and workloads. Use cost metrics enriched with tags from queries, projects, and reservations to visualize BigQuery workload costs in the context of your entire cloud bill.
+
+## Resources
+CCM allocates costs for BigQuery resources for both types of Analysis costs (on-demand and reservation-based). CCM displays costs for resources including query-level analysis, storage, and data transfer on the [**BigQuery** dashboard][1].
```

Comment:
Can we maybe add a link here to the official bigquery docs on their various pricing models
Not everyone using CCM might know the difference between the two